### PR TITLE
Finish Index Page

### PR DESCRIPTION
**Pull request by apoorvadhanala**
Finished the Index Page!

### CODE REVIEW

**Comment by yjennywang** [comment]
do you need to add the index file to this commit?

**Review by SeanT18 — APPROVED** [review]

**Comment by apoorvadhanala** [comment]
> do you need to add the index file to this commit?

Trying to add it right now - thought I had before but sorry about that!

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -10,15 +10,26 @@ body {
...
   font-family: 'Cinzel', serif;
-  color: whitesmoke;
-  font-size: 75px;
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0, 0.4); /* Black w/opacity/see-through */
+  color: white;
+  font-weight: normal;
+  font-size: 35px;
+  border: 10px solid #f1f1f1;
+  position: absolute; /* Stay fixed */
```
Comment (yjennywang) [inline src/main/webapp/style.css:27]: Depending on how much more of the fiddly position you have to do, definitely consider bootstrap at a later point:

it should be pretty easy to try out: 
https://getbootstrap.com/docs/4.5/getting-started/introduction/

And it seems particularly good for really row-based stuff and navbars: https://getbootstrap.com/docs/4.5/examples/

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
       <div>
-        <h1 class="heading-font">HOUSEHOLD</h1>
+        <h1 class="heading-font">houseHOLD</h1>
```
Comment (yjennywang) [inline src/main/webapp/index.html]: is the capitalization intentional?

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -10,15 +10,26 @@ body {
...
 .heading-font {
   font-family: 'Cinzel', serif;
-  color: rgb(237, 243, 121);
+  color: rgba(0,0,0, 0.4);
   font-size: 100px;
   text-align: center;
   
 }
 
 .body-font {
```
Comment (yjennywang) [inline src/main/webapp/style.css:19]: I'd suggest renaming the class -- this looks like it handles more than just the font.

If this is the standard font, consider moving the font-related stuff to

`body {

}`

A lot of CSS properties are inherited from the parent (it's the "cascade" in the part of the CSS name)

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
       <div>
-        <h1 class="heading-font">HOUSEHOLD</h1>
```
Comment (yjennywang) [inline src/main/webapp/index.html:14]: hmm, maybe call this "title"?

I assume this is the font that is specifically used for the title

**Review by yjennywang — APPROVED** [review]

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
```
Comment (yjennywang) [inline src/main/webapp/index.html:12]: I checked the raw content of this file against https://validator.w3.org/nu/ and there are some errors for the HTML file; can you go through and make sure the HTML is valid?

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -11,7 +11,7 @@
...
       <div>
-        <h1 class="heading-font">HOUSEHOLD</h1>
+        <h1 class="heading-font">houseHOLD</h1>
         <h3 class="body-font">Please log in here:<h3>
```
Comment (yjennywang) [inline src/main/webapp/index.html]: I think this should be a div (maybe with class=login-prompt) instead. It's not a header (in that it's not really organizing content).

**Review by yjennywang — APPROVED** [review]
approving but with comments

leaving it up to you all to figure out how/when to resolve comments without impeding your velocity :)

File: src/main/webapp/index.html (hunk 2 above)
Comment (apoorvadhanala) [reply src/main/webapp/index.html]: yeah!

File: src/main/webapp/style.css (hunk 3 above)
Comment (apoorvadhanala) [reply src/main/webapp/style.css:19]: okay that makes sense - I'll look into if we may use any other fonts and change accordingly.

**Pull request merged**